appdev/index.rst

```diff
@@ -6,16 +6,75 @@ Welcome to an open source and free platform under constant scrutiny and improvem
 Introduction
 ------------
 
-.. todo:
+Ubuntu Touch has three types of applications: Web Applications (WebApps), Scopes and Native Applications.
+Applications are packaged, distributed and deployed using a format called `click <https://click.readthedocs.io/en/latest/>`_ packaging.
+Languages of choice are QML or HTML5 for the UI and can be Javascript, Qt, C++, Python, Go for the logic.
+
+Security and app isolation
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All Ubuntu apps and scopes are confined respecting `AppArmor <https://wiki.ubuntu.com/AppArmor>`_ access control mechanism, meaning they only have access to their own resources and are isolated from other apps and parts of the system. The developer must declare which policy groups are needed for the app or scope to function properly with a `manifest.json` file.
+
+Example `manifest.json` file:
```

Comment:
[NEEDS-FIX] AppArmor profile declaration is not part of `manifest.json`, but it's a requirement of the [default hooks](https://click.readthedocs.io/en/latest/hooks.html) available for Click packages.

Could you please add an actual example of `manifest.json` that links to an AppArmor profile (e.g. for an SDK app)?